### ISSUE TITLE
NM Presentation (2075 Batch)

### ISSUE THREAD

**Issue by ErSKS** [ErSKS/NM#1]
1. Applications of NM in Science & Engineering
2. Taylor Series & it's Application
3. Curve Fitting
4. Finite Element Method
5. Gradient Descent Methods
6. Hyperparameters Optimization
7. Markov Decision Process
8. Romberg's Method
9. Runge-Kutta Methods
10. Relaxation Methods
11. Fast Fourier Transform
12. Continuous Optimization

**Comment by ErSKS** [comment]
# Applications of NM in Science & Engineering
**Team Members**

- Anup Kafle (KCE075BCT011)
- Anushil Timsina (KCE075BCT012)
- Sakar Poudel (KCE075BCT037)
- Sandeep Subedi (KCE075BCT039)

# Taylor Series & it's Application
**Team Members**

- Aayush Twayana (KCE075BCT003)
- Sameep Gaihre (KCE075BCT038)
- Sanish Shrestha (KCE075BCT041)
- Sanjog Gaihre (KCE075BCT042)

# Curve Fitting
**Team Members**

- Abhinandan Shrestha (KCE075BCT004)
- Rochak Sedai (KCE075BCT030)
- Sudip Shrestha (KCE075BCT046)
- Umesh Mahatto (KCE075BCT048)

# Finite Element Method
**Team Members**

- Bibek Manandhar (KCE075BCT013)
- Dipesh Joshi (KCE075BCT015)
- Pratirodh Sharma Pokharel (KCE075BCT026)
- Sushant Tha (KCE075BCT043)

# Gradient Descent Methods
**Team Members**

- Anish Shilpakar (KCE075BCT008)
- Anjaan Khadka (KCE075BCT009)
- Rubin Baidhya (KCE075BCT034)
- Sairush Tamang (KCE075BCT036)

# Hyperparameters Optimization
**Team Members**

- Prashant Manandhar (KCE075BCT023)
- Pratik Tamrakar (KCE075BCT025)
- Roman Shilpakar (KCE075BCT031)
- Nitesh Kumar Das (KCE075BCT022)

# Markov Decision Process
**Team Members**

- Manjil Karki (KCE075BCT019)
- Pratik Shakya (KCE075BCT024)
- Ravi Pandit (KCE075BCT028)
- Sandesh Acharya (KCE075BCT040)

# Romberg's Method
**Team Members**

- Anu Maharjan (KCE075BCT010)
- Esha Valandyo (KCE075BCT016)
- Laxmi Shrestha (KCE075BCT018)
- Rakshya Belbase (KCE075BCT027)

# Runge-Kutta Methods
**Team Members**

- Aaryashree Pokharel (KCE075BCT002)
- Nikita Pokharel (KCE075BCT021)
- Riya Parajuli (KCE075BCT029)
- Shyamron Dongol (KCE075BCT043)

# Relaxation Methods
**Team Members**

- Aabhas Dhaubanja (KCE075BCT001)
- Amit Duwal (KCE075BCT006)
- Anish Machamasi (KCE075BCT007)
- Ronish Shrestha (KCE075BCT032)

# Fast Fourier Transform
**Team Members**

- Juhel Phanju (KCE075BCT017)
- Roshan Shrestha (KCE075BCT033)
- Sachin Manandhar (KCE075BCT035)
- Sudip Manandhar (KCE075BCT045)

# Continuous Optimization
**Team Members**

- Nabin Kawan (KCE075BCT020)
- Sudeep Kasichhwa (KCE075BCT044)
- Bimal Bhoydhyo (KCE075BCT014)
- Aditya Jaishi (KCE075BCT05)
